### ISSUE TITLE
Direct mode: namespace creation log is not user-friendly

### ISSUE THREAD

**Issue by okartau** [intel/pmem-CSI#129]
In direct-nvdimm mode, there is log message logged after namespace creation using this code:
```
data, _ := ns.MarshalJSON() //nolint: gosec
glog.Infof("Namespace created: %v", data)
```
producing output:
```
I0110 glog.go:79] Namespace created: [123 34 98 108 111 99 107 100 101 118 34 58 34 112 109 101 109 48 46 49 34 44 34 100 101 118 34 58 34 110 97 109 101 115 112 97 99 101 48 46 49 34 44 34 101 110 97 98 108 101 100 34 58 102 97 108 115 101 44 34 105 100 34 58 49 44 34 109 97 112 34 58 34 100 101 118 34 44 34 109 111 100 101 34 58 34 102 115 100 97 120 34 44 34 110 97 109 101 34 58 34 56 55 54 55 53 97 54 56 45 49 52 97 49 45 49 49 101 57 45 56 49 97 102 45 53 50 53 52 48 48 97 48 97 101 51 52 34 44 34 115 105 122 101 34 58 49 48 53 52 56 54 55 52 53 54 44 34 117 117 105 100 34 58 34 56 55 54 56 54 99 50 56 45 49 52 97 49 45 49 49 101 57 45 56 49 97 102 45 53 50 53 52 48 48 97 48 97 101 51 52 34 125]
```
Seems original idea was to describe newly-created namespace, but shown message is not user-friendly

**Comment by okartau** [comment]
one-char change will fix this. Using %s instead of %v will turn msg to:
```
I0114 glog.go:79] Namespace created: {"blockdev":"pmem0.5","dev":"namespace0.5","enabled":false,"id":5,"map":"dev","mode":"fsdax","name":"1a35c26a-17cb-11e9-a186-525400a0ae34","size":1054867456,"uuid":"1a36d47e-17cb-11e9-a186-525400a0ae34"}
```

**Comment by okartau** [comment]
closing this as fix in #130 got merged
